testsuite/features/build_validation/smoke_tests/smoke_tests.template

```diff
@@ -47,6 +47,20 @@ Feature: Smoke tests for <client>
     Then I should see a "1 patch update has been scheduled for" text
     And I wait until event "Patch Update:" is completed
 
+@slemicro53_minion
+@slemicro53_ssh_minion
+@slemicro52_minion
+@slemicro52_ssh_minion
```

Comment:
It's just fine.
But we can also have a new tag for all slemicro.

Before('@slemicro') do |scenario|
```suggestion
@slemicro53_minion
@slemicro53_ssh_minion
@slemicro52_minion
@slemicro52_ssh_minion
```
  skip_this_scenario unless scenario.location.file.include? 'slemicro'
end